sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample6_AnalyzeConversation_ConversationPII_Text.md

```diff
@@ -39,45 +39,45 @@ then you can start analyzing by calling the `AnalyzeConversation`, and because t
 ## Synchronous
 
 ```C# Snippet:StartAnalyzeConversation_StartAnalayzing
-var analyzeConversationOperation = client.StartAnalyzeConversation(input, tasks);
+Operation<AnalyzeConversationJobState> analyzeConversationOperation = client.StartAnalyzeConversation(input, tasks);
 analyzeConversationOperation.WaitForCompletion();
 ```
 
 ## Asynchronous
 
 ```C# Snippet:StartAnalyzeConversationAsync_StartAnalayzing
-var analyzeConversationOperation = await client.StartAnalyzeConversationAsync(input, tasks);
+Operation<AnalyzeConversationJobState> analyzeConversationOperation = await client.StartAnalyzeConversationAsync(input, tasks);
 await analyzeConversationOperation.WaitForCompletionAsync();
 ```
 
 You can finally print the results:
 
 ```C# Snippet:StartAnalyzeConversation_ConversationPII_Text_Results
-var jobResults = analyzeConversationOperation.Value;
-foreach (var result in jobResults.Tasks.Items)
+AnalyzeConversationJobState jobResults = analyzeConversationOperation.Value;
+foreach (AnalyzeConversationJobResult result in jobResults.Tasks.Items)
 {
     var analyzeConversationPIIResult = result as AnalyzeConversationPIIResult;
 
-    var results = analyzeConversationPIIResult.Results;
+    ConversationPIIResults results = analyzeConversationPIIResult.Results;
 
     Console.WriteLine("Conversations:");
-    foreach (var conversation in results.Conversations)
+    foreach (ConversationPIIResultsConversationsItem conversation in results.Conversations)
     {
         Console.WriteLine($"Conversation #:{conversation.Id}");
         Console.WriteLine("Conversation Items: ");
-        foreach (var conversationItem in conversation.ConversationItems)
+        foreach (ConversationPIIItemResult conversationItem in conversation.ConversationItems)
```

Comment:
not for this PR, but this type seems to break the pattern of ending in "Item" -- I expected `ConversationPiiResultItem` kind of like how we have `ConversationsSummaryResultSummariesItem` in the above sample

Reply:
That's a good catch. I should add this to the list of SDK-specific renames e.g., `x-ms-client-name` I want to do to the specs repo. /cc @ChongTang

Reply:
No more models. :smile: But we should definitely keep this in mind when we do generate them later.